app/containers/Settings/index.js

```diff
@@ -121,6 +135,30 @@ class Settings extends Component {
               </View>
             </Content>
           </ScrollView>
+          <Modal
+            animationType={'slide'}
+            visible={this.state.modalVisible}
+            onRequestClose={() => this.setModalVisible(!this.state.modalVisible)}
+          >
+            <View style={{ flex: 1 }}>
+              <View style={ styles.modal }>
+                <TouchableOpacity onPress={() => this.setModalVisible(!this.state.modalVisible)}>
+                  <Icon name={'ios-arrow-dropleft'} size={24} color={'black'} style={{ padding: 10 }} />
+                </TouchableOpacity>
+              </View>
+              <WebView
+                automaticallyAdjustContentInsets={false}
+                source={{ uri: feedback }}
+                style={{ marginTop: 20 }}
+                scalesPageToFit={this.state.scalesPageToFit}
+                ref={WEBVIEW_REF}
```

Comment:
the value will always be `webview` right? then we need no variable for this, just add it here as plain string.

Reply:
i see